### PR TITLE
Vpsa news

### PR DESCRIPTION
**Pull request by cynmij**

### CODE REVIEW

**Comment by sherakama** [comment]
@cynmij,

Is this ready for review? If so please assign it to me.

File: modules/stanford_news_views/stanford_news_views.info [hunk 1]
```diff
@@ -2,7 +2,7 @@ name = Stanford News Views
 description = Views for the stanford_news module and content type.
 core = 7.x
 package = Stanford
-version = 7.x-3.1-jsvpsa
```
Comment (sherakama) [inline modules/stanford_news_views/stanford_news_views.info]: Please leave this at version 7.x-3.1-jsvpsa

File: modules/stanford_news_views/stanford_news_views.views_default.inc [hunk 2]
```diff
@@ -174,13 +174,6 @@ function stanford_news_views_views_default_views() {
   $handler->display->display_options['fields']['views_bulk_operations']['vbo_settings']['force_single'] = 0;
   $handler->display->display_options['fields']['views_bulk_operations']['vbo_settings']['entity_load_capacity'] = '10';
   $handler->display->display_options['fields']['views_bulk_operations']['vbo_operations'] = array(
-    'action::node_assign_owner_action' => array(
```
Comment (sherakama) [inline modules/stanford_news_views/stanford_news_views.views_default.inc]: Why are we removing the VBO options?
Please add them back in.

File: stanford_news.info [hunk 3]
```diff
@@ -2,7 +2,7 @@ name = Stanford News JSVPSA
 description = News Item content type, and list Views with an image and taxonomy terms
 core = 7.x
 package = Stanford
-version = 7.x-3.1-jsvpsa
+version = 7.x-3.1-jsvpsa1
```
Comment (sherakama) [inline stanford_news.info]: Please leave this at version 7.x-3.1-jsvpsa

**Comment by sherakama** [comment]
@cynmij, 

A couple of notes for you. Ball back in your court.

**Comment by cynmij** [comment]
hopefully good to go *crossing my fingers*

**Comment by sherakama** [comment]
Thanks for fixing those bits. All good now.

**Pull request merged**